packaging/makeself/makeself.sh

```diff
@@ -74,12 +74,13 @@
 # This software is released under the terms of the GNU GPL version 2 and above
```

Comment:
If these files were not written by us, them maybe we should just disable shellcheck on them?

Reply:
Could have sworn these were written by @ktsaou. If not, then yes we should just disable shellcheck for them.